packages/babel-types/src/builders/typescript/createTSUnionType.js

```diff
@@ -0,0 +1,16 @@
+import { TSUnionType } from "../generated";
+import removeTypeDuplicates from "../../modifications/typescript/removeTypeDuplicates";
+
+/**
+ * Takes an array of `types` and flattens them, removing duplicates and
+ * returns a `UnionTypeAnnotation` node containg them.
+ */
+export default function createTSUnionType(types: Array<Object>): Object {
+  const flattened = removeTypeDuplicates(types);
+
+  if (flattened.length === 1) {
+    return flattened[0];
+  } else {
+    return TSUnionType(flattened);
```

Comment:
I'm not sure if here we need to pass array, not elements of the array
But it works this way for Flow:

https://github.com/babel/babel/blob/aeebc0823415ed4770c633cedf570cf774ae89c6/packages/babel-types/src/builders/flow/createUnionTypeAnnotation.js#L9-L19

Reply:
Yeah, only 1 parameter is accepted:

```
TSUnionType: Too many arguments passed. Received 2 but can receive no more than 1
```